benchmark/CMakeLists.txt

```diff
@@ -148,7 +149,7 @@ add_umf_benchmark(
     NAME benchmark
     SRCS benchmark.cpp
     LIBS ${LIBS_OPTIONAL} benchmark::benchmark
-    LIBDIRS ${LIB_DIRS})
+    LIBDIRS ${LIB_DIRS} EXTRARGS --benchmark_filter=threads:1$)
```

Comment:
please add a comment "limit running benchmarks in CI tests to single-threaded"

Reply:
Done